packages/platform-core/API.md

```diff
@@ -29,13 +30,57 @@ export enum CDKContextKey {
     DEPLOYMENT_TYPE = "amplify-backend-type"
 }
 
+// @public
+export class CwdPackageJsonLoader {
+    read(): {
+        name?: string | undefined;
+        version?: string | undefined;
+        type?: "module" | "commonjs" | undefined;
+    };

```

Comment:
Should this use `type PackageJson` just defined below?
Or is it some api extractor behavior with zod?

Reply:
updated